src/frontend/src/components/pages/CSSAPI/index.ts

```diff
@@ -0,0 +1,23 @@
+import { GetMultipleRolesCard } from "./GetMultipleRolesCard";
```

Comment:
```suggestion
export * from "./GetMultipleRolesCard";
```
You just have to do exports like this. No need to import, then export.